clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -659,95 +749,50 @@ public void testPollTimerExpirationShouldNotMarkMemberStaleIfMemberAlreadyLeavin
 
     @Test
     public void testisExpiredByUsedForLogging() {
-        Timer pollTimer = spy(time.timer(DEFAULT_MAX_POLL_INTERVAL_MS));
-        heartbeatRequestManager = new HeartbeatRequestManager(new LogContext(), pollTimer, config(),
-            coordinatorRequestManager, membershipManager, heartbeatState, heartbeatRequestState,
-            backgroundEventHandler, metrics);
         when(membershipManager.shouldSkipHeartbeat()).thenReturn(false);
 
         int exceededTimeMs = 5;
         time.sleep(DEFAULT_MAX_POLL_INTERVAL_MS + exceededTimeMs);
 
+        when(membershipManager.isLeavingGroup()).thenReturn(false);
         NetworkClientDelegate.PollResult pollResult = heartbeatRequestManager.poll(time.milliseconds());
         assertEquals(1, pollResult.unsentRequests.size());
         verify(membershipManager).transitionToSendingLeaveGroup(true);
         verify(pollTimer, never()).isExpiredBy();
-        assertEquals(exceededTimeMs, pollTimer.isExpiredBy());
 
         clearInvocations(pollTimer);
         heartbeatRequestManager.resetPollTimer(time.milliseconds());
         verify(pollTimer).isExpiredBy();
     }
 
-    @Test
-    public void testHeartbeatMetrics() {
```

Comment:
just to double check, we're removing this because it's covered in the `HeartbeatMetricsManangerTest` right?

Reply:
Yes, you are correct